content/en-us/ui/text-filtering.md

```diff
@@ -105,8 +105,8 @@ local function getTextObject(message, fromPlayerId)
 	end)
 	if success then
 		return textObject
-	elseif errorMessage then
-		print("Error generating TextFilterResult:", errorMessage)
+	else
```

Comment:
this `else` isnt necessary as the function returns within the if statement

Reply:
I've removed it.